usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1430,6 +1433,7 @@
                 ("prime_awardee_country_name", "broker_subaward__legal_entity_country_name"),
                 ("prime_awardee_address_line_1", "broker_subaward__legal_entity_address_line1"),
                 ("prime_awardee_city_name", "broker_subaward__legal_entity_city_name"),
+                ("prime_awardee_county_name", "award__latest_transaction__assistance_data__legal_entity_county_name"),
```

Comment:
Would be nice to add this column to `broker_subaward` unless of course it isn't on the broker table that `broker_subaward` is suppose to be an exact copy of.

Reply:
Yes, sorry if that was unclear, Broker does not have this data either, which is why we are pulling from the prime award.

Reply:
Oh well that stinks. Ignore my comment on approval then. 👍